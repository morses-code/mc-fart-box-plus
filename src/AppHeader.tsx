import React from "react";

const AppHeader: React.FC = () => {
  return <h1 className="app-header">Caleb's Fart Box</h1>;
}

export default AppHeader;