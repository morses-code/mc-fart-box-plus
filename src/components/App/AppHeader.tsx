/**
 * @file AppHeader.js
 * @description Contains the AppHeader component for the header section of the app.
 */

import React from 'react';

/**
 * @component AppHeader
 * @description A component that represents the header section of the app.
 * @return {JSX.Element} The rendered AppHeader component.
 */
const AppHeader: React.FC = () => {
	return (
		<div>
			<img
				src='/logo-no-background.png'
				alt='Logo'
				className='header-logo'
			/>
		</div>
	);
};

export default AppHeader;
