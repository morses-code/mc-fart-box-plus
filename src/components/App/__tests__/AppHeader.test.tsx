import React from 'react';
import { render } from '@testing-library/react';
import AppHeader from '../AppHeader';

describe('AppHeader', () => {
	it('renders correctly and matches snapshot', () => {
		const { container } = render(<AppHeader />);

		expect(container.firstChild).toMatchSnapshot();
	});
});
