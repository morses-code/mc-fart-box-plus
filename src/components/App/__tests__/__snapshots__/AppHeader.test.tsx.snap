// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader renders correctly and matches snapshot 1`] = `
<div>
  <img
    alt="Logo"
    class="header-logo"
    src="/logo-no-background.png"
  />
</div>
`;
